Src/CSharpier.Cli.Tests/CliTests.cs

```diff
@@ -315,6 +315,47 @@ public async Task Should_Not_Fail_On_Bad_Csproj()
         result.Output.Should().StartWith("Warning The csproj at ");
     }
 
+    [Test]
+    public async Task Should_Not_Fail_On_Mismatched_MSBuild_With_No_Check()
+    {
+        await this.WriteFileAsync(
+            "Test.csproj",
+            @"<Project Sdk=""Microsoft.NET.Sdk"">
```

Comment:
Since the contents of these files are static, you should consider creating them as actual files and committing them.
Then you wouldn't need to escape all the ""`